gql/resolver.go

```diff
@@ -570,7 +572,9 @@ func (dr *dealResolver) message(ctx context.Context, checkpoint dealcheckpoints.
 	case dealcheckpoints.Transferred:
 		return "Ready to Publish"
 	case dealcheckpoints.Published:
-		return "Awaiting Publish Confirmation"
+		elapsedEpochs := uint64(time.Since(checkpointAt).Seconds()) / build.BlockDelaySecs
+		confidenceEpochs := build.MessageConfidence * 2
+		return fmt.Sprintf("Awaiting Publish Confirmation (%d/%d)", elapsedEpochs, confidenceEpochs)
```

Comment:
I'd suggest adding the word "epochs" so that the provider understands what 2/10 refers to
```suggestion
		return fmt.Sprintf("Awaiting Publish Confirmation (%d/%d epochs)", elapsedEpochs, confidenceEpochs)
```